include/Sugar_Smarty.php

```diff
@@ -151,10 +151,10 @@ public function fetch($resource_name, $cache_id = null, $compile_id = null, $dis
         $state = new SuiteCRM\StateSaver();
         $state->pushErrorLevel('sugar_smarty_errors');
         
-        if (!(isset($sugar_config['developerMode']) && $sugar_config['developerMode'])) {
-            $level = isset($sugar_config['smarty_error_level']) ? $sugar_config['smarty_error_level'] : 0;
-            error_reporting($level);
-        }
+
```

Comment:
Not relevant?

Reply:
This  decouples the error messaging from debug mode. When you need to see the cached tpl, development mode will clear them. So this helps to see the errors.